_meeting-notes/2019-12-03.md

```diff
@@ -139,6 +139,10 @@ Full list can be at https://github.com/ipfs/testground/issues/196
 
 - ==@raulk==
     - Review daemon PR.
+    - (HIGH PRIO) Outline candidate solutions and trade-offs for libp2p deep traceability.
+    - Initiate SPEC overhaul to align with actual implementation.
+    - Error handling guidelines.
+    - Start conceptualising multiple TB transfer test case design. 
```

Comment:
Ah! Read this one better now. It is not necessarily the TB transfer (@dirkmc is on top of that and @hacdias is on top of the adding part with #58). It is "what to do to have the multiple TB Harddrive so that we can run the test in AWS/locally, etc" @raulk